examples/main/layer-examples.js

```diff
@@ -160,6 +160,20 @@ const ExtrudedChoroplethLayer64Example = {
   }
 };
 
+const HexagonLayerExample = {
+  layer: HexagonLayer,
+  props: {
+    id: 'hexagonLayer',
+    data: dataSamples.hexagons,
+    hexagonVertices: dataSamples.hexagons[0].vertices,
+    getColor: h => [48, 128, h.value * 255],
+    getElevation: h => h.value,
+    enable3d: true,
```

Comment:
Note for API audit: we should define the name of this prop across layers (`enable3d` vs. `extruded` vs...) and define expected semantics (generate simpler geometry, turn off lighting etc)